ui/app/adapters/secret-v2-version.js

```diff
@@ -54,6 +54,12 @@ export default ApplicationAdapter.extend({
     });
   },
 
+  querySecretDataByVersion(id) {
```

Comment:
Should we return the error so we have access to the secret metadata that tells us if a secret has been deleted/destroyed? Right now the promise fails if a secret version has been deleted/destroyed.